examples/isolated-objects/isolated_objects.bal

```diff
@@ -0,0 +1,86 @@
+import ballerina/io;
+
+type Coordinates record {|
+    decimal latitude;
+    decimal longitude;
+|};
+
+isolated class Cities {
+
+    final string country;
+
+    // The initialization expression of a field of an
+    // `isolated` object has to be an `isolated` expression,
+    // which itself will be an isolated root.
```

Comment:
```suggestion
    // which itself will be an `isolated` root.
```

Reply:
"isolated root" and "isolated expression" are concepts that do not have a direct representation in the code, unlike `isolated` objects and variables.

Applicable to other suggestions for the same too.